lib/PuppeteerSharp/Browser.cs

```diff
@@ -68,6 +69,10 @@ public string WebSocketEndpoint
             return await target.Page();
         }
 
+        public async Task<IEnumerable<Page>> Pages() =>
+            (await Task.WhenAll(this._targets.Select(x => x.Value.Page())))
+                .Where(x => x != null);
```

Comment:
No problem! I didn't check how it was used elsewhere!

Reply:
```csharp
        public void Disconnect() => Connection.Dispose();
```

Should this also be two lines?

And the following three or two lines?

```csharp
        public async Task<IEnumerable<Page>> Pages()
            => (await Task.WhenAll(this._targets.Select(x => x.Value.Page()))).Where(x => x != null);
```

Reply:
I can gladly do https://github.com/kblok/puppeteer-sharp/issues/130 in another PR! I've already set up the StyleCop analyzer on multiple projects, so I can make sure it follows your style!

Reply:
I have a column gridline at 120 columns. so that would be our line limit.
I started to read about StyleCop or editorconfig, the thing is that I'm on Visual Studio for mac, so we need something cross IDE.

Reply:
Analyzers are cross-IDE since they hook into the compilation process. I had an intern use VS for Mac and it worked great for him!